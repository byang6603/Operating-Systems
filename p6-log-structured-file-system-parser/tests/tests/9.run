../bin/parse_lfs ls ../sample_files/b.img
