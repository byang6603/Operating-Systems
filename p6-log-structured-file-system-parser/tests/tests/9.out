/
         256 1986-11-24 09:03:49 UTC -rw-r-----    bar.txt 
           2 1979-07-10 18:33:15 UTC -rw-r-----    foo.txt 
        4990 1992-09-29 15:56:15 UTC drwxr-xr-x    dir     
           1 1978-03-27 14:05:25 UTC -rwxr-xr-x        file_0.txt
           8 1971-08-22 08:54:33 UTC -rwxr-xr-x        file_1.txt
          16 1980-12-01 14:14:01 UTC -rwxr-xr-x        file_10.txt
           8 1991-02-20 19:57:14 UTC -rwxr-xr-x        file_100.txt
        1024 1991-02-15 19:28:56 UTC -rwxr-xr-x        file_101.txt
           8 1988-08-31 12:38:32 UTC -rwxr-xr-x        file_102.txt
           2 1983-06-02 03:15:27 UTC -rwxr-xr-x        file_103.txt
         512 1995-06-06 15:00:30 UTC -rwxr-xr-x        file_104.txt
           1 1991-09-10 02:35:17 UTC -rwxr-xr-x        file_105.txt
           1 1985-01-04 21:05:40 UTC -rwxr-xr-x        file_106.txt
         128 2000-05-13 02:37:13 UTC -rwxr-xr-x        file_107.txt
           4 1975-06-28 09:43:22 UTC -rwxr-xr-x        file_108.txt
         128 1984-07-02 14:52:00 UTC -rwxr-xr-x        file_109.txt
          32 1981-05-28 23:42:00 UTC -rwxr-xr-x        file_11.txt
        2048 1996-05-13 09:04:26 UTC -rwxr-xr-x        file_110.txt
        2048 2003-12-17 04:23:55 UTC -rwxr-xr-x        file_111.txt
           2 1999-10-02 02:27:04 UTC -rwxr-xr-x        file_112.txt
         128 1996-01-02 21:49:47 UTC -rwxr-xr-x        file_113.txt
         128 1975-08-18 02:20:46 UTC -rwxr-xr-x        file_114.txt
         512 1986-06-10 02:09:17 UTC -rwxr-xr-x        file_115.txt
        2048 1989-10-20 23:44:41 UTC -rwxr-xr-x        file_116.txt
          16 2001-07-01 03:32:57 UTC -rwxr-xr-x        file_117.txt
           8 1995-04-11 04:52:15 UTC -rwxr-xr-x        file_118.txt
          16 1986-12-15 06:48:43 UTC -rwxr-xr-x        file_119.txt
           1 1983-02-26 06:28:34 UTC -rwxr-xr-x        file_12.txt
        1024 1986-09-30 16:37:39 UTC -rwxr-xr-x        file_120.txt
          16 1986-12-31 10:27:14 UTC -rwxr-xr-x        file_121.txt
        2048 1979-07-10 06:28:14 UTC -rwxr-xr-x        file_122.txt
         256 1995-05-27 00:36:36 UTC -rwxr-xr-x        file_123.txt
          16 1998-08-21 12:05:04 UTC -rwxr-xr-x        file_124.txt
          32 1995-03-26 08:41:11 UTC -rwxr-xr-x        file_125.txt
        1024 1995-11-25 17:38:54 UTC -rwxr-xr-x        file_126.txt
         256 1972-08-27 22:04:36 UTC -rwxr-xr-x        file_127.txt
          16 1986-10-01 03:25:12 UTC -rwxr-xr-x        file_128.txt
           4 2001-03-05 17:10:05 UTC -rwxr-xr-x        file_129.txt
          32 1982-08-21 14:49:57 UTC -rwxr-xr-x        file_13.txt
           2 2002-07-02 00:25:27 UTC -rwxr-xr-x        file_130.txt
        2048 1973-11-11 09:45:22 UTC -rwxr-xr-x        file_131.txt
           4 1988-04-10 23:01:57 UTC -rwxr-xr-x        file_132.txt
          64 1987-02-20 23:26:55 UTC -rwxr-xr-x        file_133.txt
         256 1987-04-12 13:00:49 UTC -rwxr-xr-x        file_134.txt
          16 1979-09-06 00:25:23 UTC -rwxr-xr-x        file_135.txt
           8 1974-10-21 12:25:08 UTC -rwxr-xr-x        file_136.txt
           2 2002-04-16 10:06:29 UTC -rwxr-xr-x        file_137.txt
        2048 1976-01-07 13:25:33 UTC -rwxr-xr-x        file_138.txt
         512 1980-04-16 22:08:30 UTC -rwxr-xr-x        file_139.txt
          64 1982-03-06 16:31:22 UTC -rwxr-xr-x        file_14.txt
          16 1982-10-07 11:21:25 UTC -rwxr-xr-x        file_140.txt
         512 2002-05-19 22:29:59 UTC -rwxr-xr-x        file_141.txt
          32 1980-03-23 12:55:21 UTC -rwxr-xr-x        file_142.txt
           2 1978-06-29 00:11:26 UTC -rwxr-xr-x        file_143.txt
         256 1994-05-12 06:42:42 UTC -rwxr-xr-x        file_144.txt
          64 1985-11-07 15:16:08 UTC -rwxr-xr-x        file_145.txt
         128 1994-12-06 02:20:43 UTC -rwxr-xr-x        file_146.txt
         256 1980-02-20 16:50:19 UTC -rwxr-xr-x        file_147.txt
          32 1983-04-29 05:12:01 UTC -rwxr-xr-x        file_148.txt
         512 1986-03-06 17:35:54 UTC -rwxr-xr-x        file_149.txt
           2 1970-08-07 03:09:20 UTC -rwxr-xr-x        file_15.txt
          16 1997-02-02 23:39:03 UTC -rwxr-xr-x        file_150.txt
         512 2000-01-26 21:49:40 UTC -rwxr-xr-x        file_151.txt
           8 2003-03-06 04:03:08 UTC -rwxr-xr-x        file_152.txt
         128 1972-08-02 16:30:13 UTC -rwxr-xr-x        file_153.txt
           2 1991-06-12 08:49:12 UTC -rwxr-xr-x        file_154.txt
        1024 1997-10-15 02:31:09 UTC -rwxr-xr-x        file_155.txt
          64 1997-10-26 01:11:25 UTC -rwxr-xr-x        file_156.txt
          32 1983-04-27 12:51:02 UTC -rwxr-xr-x        file_157.txt
         128 2000-06-11 00:35:45 UTC -rwxr-xr-x        file_158.txt
           1 1980-07-16 14:59:33 UTC -rwxr-xr-x        file_159.txt
           1 1992-08-08 03:01:09 UTC -rwxr-xr-x        file_16.txt
        2048 1980-06-20 16:24:03 UTC -rwxr-xr-x        file_160.txt
         256 1998-12-01 11:24:08 UTC -rwxr-xr-x        file_161.txt
           4 1984-05-27 00:44:55 UTC -rwxr-xr-x        file_162.txt
         512 1998-09-29 15:26:01 UTC -rwxr-xr-x        file_163.txt
        1024 1982-01-11 21:13:59 UTC -rwxr-xr-x        file_164.txt
         512 2001-09-05 13:45:44 UTC -rwxr-xr-x        file_165.txt
        2048 1974-05-26 02:14:20 UTC -rwxr-xr-x        file_166.txt
         128 1986-11-01 09:39:07 UTC -rwxr-xr-x        file_167.txt
           2 1999-12-11 10:15:09 UTC -rwxr-xr-x        file_168.txt
           4 1980-05-31 15:39:53 UTC -rwxr-xr-x        file_169.txt
         128 1994-06-17 16:22:23 UTC -rwxr-xr-x        file_17.txt
        1024 1997-02-15 07:47:37 UTC -rwxr-xr-x        file_170.txt
         512 1978-09-07 07:59:31 UTC -rwxr-xr-x        file_171.txt
           4 1978-10-09 00:32:49 UTC -rwxr-xr-x        file_172.txt
           2 1973-04-29 07:05:55 UTC -rwxr-xr-x        file_173.txt
          16 1987-03-05 08:10:57 UTC -rwxr-xr-x        file_174.txt
           8 2003-02-17 07:15:31 UTC -rwxr-xr-x        file_175.txt
           2 1989-03-05 22:22:03 UTC -rwxr-xr-x        file_176.txt
           2 1978-01-28 20:54:36 UTC -rwxr-xr-x        file_177.txt
         256 1979-03-30 10:28:47 UTC -rwxr-xr-x        file_178.txt
          64 2002-07-02 03:34:04 UTC -rwxr-xr-x        file_179.txt
        1024 1999-11-16 04:44:02 UTC -rwxr-xr-x        file_18.txt
        1024 1994-04-03 14:30:31 UTC -rwxr-xr-x        file_180.txt
           1 1972-04-22 20:30:46 UTC -rwxr-xr-x        file_181.txt
         512 1998-07-18 11:46:40 UTC -rwxr-xr-x        file_182.txt
         512 1993-05-28 04:56:35 UTC -rwxr-xr-x        file_183.txt
        1024 1974-11-23 13:00:59 UTC -rwxr-xr-x        file_184.txt
         128 1985-12-18 06:58:48 UTC -rwxr-xr-x        file_185.txt
        1024 1987-03-02 17:50:40 UTC -rwxr-xr-x        file_186.txt
          16 2002-09-17 14:12:24 UTC -rwxr-xr-x        file_187.txt
         256 1999-04-13 19:49:51 UTC -rwxr-xr-x        file_188.txt
         512 1983-08-02 04:49:21 UTC -rwxr-xr-x        file_189.txt
           2 1987-11-12 17:52:39 UTC -rwxr-xr-x        file_19.txt
          16 1979-03-24 15:34:53 UTC -rwxr-xr-x        file_190.txt
           8 1975-09-22 22:36:50 UTC -rwxr-xr-x        file_191.txt
           4 1978-06-23 02:36:25 UTC -rwxr-xr-x        file_192.txt
           2 1993-08-17 16:19:48 UTC -rwxr-xr-x        file_193.txt
         128 1970-06-12 00:25:47 UTC -rwxr-xr-x        file_194.txt
          16 1990-07-03 23:50:24 UTC -rwxr-xr-x        file_195.txt
           1 1991-04-13 16:28:28 UTC -rwxr-xr-x        file_196.txt
         128 1974-11-04 02:40:08 UTC -rwxr-xr-x        file_197.txt
         256 1973-04-24 19:52:27 UTC -rwxr-xr-x        file_198.txt
         256 1987-03-14 13:06:34 UTC -rwxr-xr-x        file_199.txt
         512 1976-06-30 23:58:32 UTC -rwxr-xr-x        file_2.txt
           2 1982-07-29 11:27:38 UTC -rwxr-xr-x        file_20.txt
        1024 1985-04-03 18:20:01 UTC -rwxr-xr-x        file_200.txt
          16 2000-06-09 03:40:05 UTC -rwxr-xr-x        file_201.txt
           4 1995-11-18 21:06:05 UTC -rwxr-xr-x        file_202.txt
          16 1994-01-09 18:52:50 UTC -rwxr-xr-x        file_203.txt
          32 2003-10-06 10:46:00 UTC -rwxr-xr-x        file_204.txt
           8 1979-01-11 15:39:58 UTC -rwxr-xr-x        file_205.txt
         256 1993-02-16 12:31:18 UTC -rwxr-xr-x        file_206.txt
        1024 1988-11-29 19:30:59 UTC -rwxr-xr-x        file_207.txt
          64 1987-02-08 12:34:34 UTC -rwxr-xr-x        file_208.txt
           2 2002-05-15 23:00:05 UTC -rwxr-xr-x        file_209.txt
        1024 1970-08-08 08:35:04 UTC -rwxr-xr-x        file_21.txt
          32 1987-05-22 09:27:59 UTC -rwxr-xr-x        file_210.txt
         512 1977-05-02 13:28:01 UTC -rwxr-xr-x        file_211.txt
           1 1970-08-27 05:53:47 UTC -rwxr-xr-x        file_212.txt
          32 1981-11-27 07:37:35 UTC -rwxr-xr-x        file_213.txt
         512 2000-09-26 18:24:37 UTC -rwxr-xr-x        file_214.txt
           2 1975-07-19 18:54:46 UTC -rwxr-xr-x        file_215.txt
          64 1997-11-13 14:36:23 UTC -rwxr-xr-x        file_216.txt
        1024 1983-11-17 22:38:13 UTC -rwxr-xr-x        file_217.txt
         256 1974-03-26 19:30:07 UTC -rwxr-xr-x        file_218.txt
           4 1993-02-14 20:49:10 UTC -rwxr-xr-x        file_219.txt
           4 1993-09-24 21:32:31 UTC -rwxr-xr-x        file_22.txt
           2 1997-06-18 03:27:35 UTC -rwxr-xr-x        file_220.txt
          32 1983-06-17 11:05:00 UTC -rwxr-xr-x        file_221.txt
         512 1998-11-06 19:26:01 UTC -rwxr-xr-x        file_222.txt
         256 1971-11-29 16:26:56 UTC -rwxr-xr-x        file_223.txt
           4 1973-01-22 13:47:45 UTC -rwxr-xr-x        file_224.txt
           8 1999-04-17 19:51:48 UTC -rwxr-xr-x        file_225.txt
           8 1992-05-31 16:17:21 UTC -rwxr-xr-x        file_226.txt
          64 1994-05-05 06:16:13 UTC -rwxr-xr-x        file_227.txt
        2048 1970-02-09 08:54:52 UTC -rwxr-xr-x        file_228.txt
           4 1995-09-23 12:09:48 UTC -rwxr-xr-x        file_229.txt
          64 1994-07-18 20:28:30 UTC -rwxr-xr-x        file_23.txt
           8 1977-07-07 05:45:43 UTC -rwxr-xr-x        file_230.txt
        1024 1985-05-13 03:14:54 UTC -rwxr-xr-x        file_231.txt
         512 1992-02-21 02:12:49 UTC -rwxr-xr-x        file_232.txt
          16 2003-05-25 02:51:48 UTC -rwxr-xr-x        file_233.txt
           8 1975-05-13 08:30:40 UTC -rwxr-xr-x        file_234.txt
           4 1991-11-16 23:21:45 UTC -rwxr-xr-x        file_235.txt
          64 1978-05-26 04:54:42 UTC -rwxr-xr-x        file_236.txt
         512 1998-06-28 21:01:58 UTC -rwxr-xr-x        file_237.txt
           8 1976-10-06 05:15:40 UTC -rwxr-xr-x        file_238.txt
         512 1995-07-03 17:29:17 UTC -rwxr-xr-x        file_239.txt
          32 1978-12-06 07:07:48 UTC -rwxr-xr-x        file_24.txt
          32 1996-11-01 06:24:59 UTC -rwxr-xr-x        file_240.txt
          16 1994-02-24 14:43:39 UTC -rwxr-xr-x        file_241.txt
          16 2002-11-02 06:57:18 UTC -rwxr-xr-x        file_242.txt
          32 1997-06-27 12:18:46 UTC -rwxr-xr-x        file_243.txt
         512 1972-01-12 08:44:10 UTC -rwxr-xr-x        file_244.txt
          16 1999-07-20 11:44:51 UTC -rwxr-xr-x        file_245.txt
           1 2003-01-13 07:13:33 UTC -rwxr-xr-x        file_246.txt
           8 1999-11-24 23:20:34 UTC -rwxr-xr-x        file_247.txt
          16 1979-05-27 20:46:01 UTC -rwxr-xr-x        file_248.txt
           8 1973-03-29 13:06:36 UTC -rwxr-xr-x        file_249.txt
          32 1986-09-07 01:16:32 UTC -rwxr-xr-x        file_25.txt
           8 1988-12-30 06:32:40 UTC -rwxr-xr-x        file_250.txt
          16 1972-11-02 10:36:32 UTC -rwxr-xr-x        file_251.txt
         512 1986-09-13 00:11:36 UTC -rwxr-xr-x        file_252.txt
        2048 1983-10-27 12:21:37 UTC -rwxr-xr-x        file_253.txt
           4 1974-10-01 03:03:28 UTC -rwxr-xr-x        file_254.txt
          16 1989-10-04 13:59:21 UTC -rwxr-xr-x        file_255.txt
           1 1979-02-01 18:36:22 UTC -rwxr-xr-x        file_256.txt
         512 1997-02-28 19:20:49 UTC -rwxr-xr-x        file_257.txt
           2 1980-01-28 06:38:31 UTC -rwxr-xr-x        file_258.txt
        2048 1979-03-13 03:31:14 UTC -rwxr-xr-x        file_259.txt
        2048 1979-03-12 15:59:35 UTC -rwxr-xr-x        file_26.txt
          32 1988-11-11 17:53:34 UTC -rwxr-xr-x        file_260.txt
        2048 1987-08-03 12:24:14 UTC -rwxr-xr-x        file_261.txt
        1024 1994-07-23 06:46:08 UTC -rwxr-xr-x        file_262.txt
          16 1976-12-23 06:29:19 UTC -rwxr-xr-x        file_263.txt
           2 1986-12-16 01:38:59 UTC -rwxr-xr-x        file_264.txt
         512 1999-12-02 15:16:49 UTC -rwxr-xr-x        file_265.txt
           4 1998-11-08 05:51:05 UTC -rwxr-xr-x        file_266.txt
          64 1995-05-10 06:33:41 UTC -rwxr-xr-x        file_267.txt
         128 1994-05-20 22:41:43 UTC -rwxr-xr-x        file_268.txt
        2048 1971-08-04 21:29:41 UTC -rwxr-xr-x        file_269.txt
         256 1977-04-03 01:28:24 UTC -rwxr-xr-x        file_27.txt
        2048 1986-10-31 10:25:54 UTC -rwxr-xr-x        file_270.txt
         512 1987-03-12 15:29:39 UTC -rwxr-xr-x        file_271.txt
          32 1995-09-28 12:13:21 UTC -rwxr-xr-x        file_272.txt
           1 1985-08-23 03:46:09 UTC -rwxr-xr-x        file_273.txt
          64 1980-08-27 14:11:21 UTC -rwxr-xr-x        file_274.txt
           4 1997-10-08 20:57:31 UTC -rwxr-xr-x        file_275.txt
          64 1981-03-02 01:53:56 UTC -rwxr-xr-x        file_276.txt
         256 1979-08-31 07:47:50 UTC -rwxr-xr-x        file_277.txt
        1024 1993-08-23 06:41:01 UTC -rwxr-xr-x        file_278.txt
           2 1990-07-26 22:39:57 UTC -rwxr-xr-x        file_279.txt
          32 1998-10-17 03:05:25 UTC -rwxr-xr-x        file_28.txt
           1 1982-11-26 20:54:26 UTC -rwxr-xr-x        file_280.txt
        2048 1978-08-12 23:36:38 UTC -rwxr-xr-x        file_281.txt
           4 1993-05-28 09:16:29 UTC -rwxr-xr-x        file_282.txt
           8 1999-08-08 21:06:03 UTC -rwxr-xr-x        file_283.txt
           1 1992-06-07 11:58:15 UTC -rwxr-xr-x        file_284.txt
          32 1998-02-25 12:19:58 UTC -rwxr-xr-x        file_285.txt
          64 1985-05-02 21:28:20 UTC -rwxr-xr-x        file_286.txt
          16 2001-07-09 06:34:37 UTC -rwxr-xr-x        file_287.txt
         128 1991-04-16 18:03:43 UTC -rwxr-xr-x        file_288.txt
           4 1995-05-30 04:06:51 UTC -rwxr-xr-x        file_289.txt
         256 1996-02-03 01:03:24 UTC -rwxr-xr-x        file_29.txt
         256 1976-09-08 20:28:48 UTC -rwxr-xr-x        file_290.txt
           8 1976-02-16 22:20:13 UTC -rwxr-xr-x        file_291.txt
         512 1978-12-21 02:54:02 UTC -rwxr-xr-x        file_292.txt
          16 2001-03-31 03:14:56 UTC -rwxr-xr-x        file_293.txt
           2 1983-02-08 04:49:33 UTC -rwxr-xr-x        file_294.txt
         512 1995-12-05 04:33:01 UTC -rwxr-xr-x        file_295.txt
           4 1997-02-20 04:54:41 UTC -rwxr-xr-x        file_296.txt
          32 1977-12-06 21:03:34 UTC -rwxr-xr-x        file_297.txt
          64 1987-04-03 21:29:38 UTC -rwxr-xr-x        file_298.txt
         256 1987-07-01 13:59:21 UTC -rwxr-xr-x        file_299.txt
         512 1977-07-07 11:59:11 UTC -rwxr-xr-x        file_3.txt
           2 1999-12-31 17:24:40 UTC -rwxr-xr-x        file_30.txt
           8 1973-01-01 03:33:46 UTC -rwxr-xr-x        file_31.txt
        1024 1997-09-23 09:57:57 UTC -rwxr-xr-x        file_32.txt
        2048 1972-06-21 03:46:08 UTC -rwxr-xr-x        file_33.txt
          32 1980-07-07 15:32:57 UTC -rwxr-xr-x        file_34.txt
        1024 1990-09-18 17:11:45 UTC -rwxr-xr-x        file_35.txt
        1024 1982-12-05 21:25:33 UTC -rwxr-xr-x        file_36.txt
           2 1995-05-25 04:45:30 UTC -rwxr-xr-x        file_37.txt
          32 1989-10-15 19:05:06 UTC -rwxr-xr-x        file_38.txt
           4 1981-01-27 13:28:52 UTC -rwxr-xr-x        file_39.txt
           8 1997-01-04 20:50:52 UTC -rwxr-xr-x        file_4.txt
         256 1971-01-18 21:10:29 UTC -rwxr-xr-x        file_40.txt
           2 2000-09-15 09:19:07 UTC -rwxr-xr-x        file_41.txt
         256 1992-06-24 13:10:52 UTC -rwxr-xr-x        file_42.txt
        2048 1984-03-15 03:39:04 UTC -rwxr-xr-x        file_43.txt
         512 1979-04-27 10:32:00 UTC -rwxr-xr-x        file_44.txt
           8 1970-08-19 16:05:10 UTC -rwxr-xr-x        file_45.txt
          64 1984-10-19 06:48:24 UTC -rwxr-xr-x        file_46.txt
          16 2001-12-02 13:33:10 UTC -rwxr-xr-x        file_47.txt
        1024 1995-02-03 08:27:33 UTC -rwxr-xr-x        file_48.txt
         256 1980-08-24 21:55:23 UTC -rwxr-xr-x        file_49.txt
         256 1980-06-16 17:39:25 UTC -rwxr-xr-x        file_5.txt
         128 1985-10-04 17:48:45 UTC -rwxr-xr-x        file_50.txt
         128 1973-08-22 06:18:07 UTC -rwxr-xr-x        file_51.txt
        2048 1981-04-01 06:30:27 UTC -rwxr-xr-x        file_52.txt
         512 1975-06-20 01:44:12 UTC -rwxr-xr-x        file_53.txt
           8 1998-03-09 02:46:37 UTC -rwxr-xr-x        file_54.txt
           8 1990-03-06 13:38:15 UTC -rwxr-xr-x        file_55.txt
        1024 1992-02-24 03:00:44 UTC -rwxr-xr-x        file_56.txt
           1 1973-05-09 05:09:08 UTC -rwxr-xr-x        file_57.txt
         256 1997-06-06 15:06:39 UTC -rwxr-xr-x        file_58.txt
          32 1986-11-30 16:29:05 UTC -rwxr-xr-x        file_59.txt
         128 1984-11-17 13:12:33 UTC -rwxr-xr-x        file_6.txt
           8 1999-06-11 06:12:32 UTC -rwxr-xr-x        file_60.txt
           8 1993-05-27 18:54:15 UTC -rwxr-xr-x        file_61.txt
          16 1989-11-30 20:02:51 UTC -rwxr-xr-x        file_62.txt
          16 1993-02-22 02:33:26 UTC -rwxr-xr-x        file_63.txt
           8 1995-11-15 22:40:24 UTC -rwxr-xr-x        file_64.txt
           2 2000-06-06 11:35:49 UTC -rwxr-xr-x        file_65.txt
         256 1979-11-01 06:08:07 UTC -rwxr-xr-x        file_66.txt
          16 1974-10-11 06:28:53 UTC -rwxr-xr-x        file_67.txt
         512 1991-10-20 02:44:15 UTC -rwxr-xr-x        file_68.txt
          16 1999-08-16 01:13:13 UTC -rwxr-xr-x        file_69.txt
           4 2003-02-06 15:30:24 UTC -rwxr-xr-x        file_7.txt
           2 1985-11-06 19:57:45 UTC -rwxr-xr-x        file_70.txt
           4 1992-11-05 23:54:45 UTC -rwxr-xr-x        file_71.txt
          64 1996-04-20 20:55:17 UTC -rwxr-xr-x        file_72.txt
          32 1974-04-21 19:31:33 UTC -rwxr-xr-x        file_73.txt
         512 1973-01-09 13:56:45 UTC -rwxr-xr-x        file_74.txt
         512 1971-08-06 17:50:13 UTC -rwxr-xr-x        file_75.txt
          64 1974-12-08 11:36:43 UTC -rwxr-xr-x        file_76.txt
           1 1987-10-28 20:45:09 UTC -rwxr-xr-x        file_77.txt
        2048 2003-07-08 07:23:23 UTC -rwxr-xr-x        file_78.txt
         256 2000-01-10 20:04:17 UTC -rwxr-xr-x        file_79.txt
         512 2002-03-04 05:40:22 UTC -rwxr-xr-x        file_8.txt
         512 1998-06-21 18:40:32 UTC -rwxr-xr-x        file_80.txt
         512 1985-04-01 11:35:04 UTC -rwxr-xr-x        file_81.txt
        1024 2003-09-01 02:22:24 UTC -rwxr-xr-x        file_82.txt
         128 1975-09-11 11:33:55 UTC -rwxr-xr-x        file_83.txt
          64 1990-09-18 13:19:17 UTC -rwxr-xr-x        file_84.txt
          64 1997-10-28 15:31:58 UTC -rwxr-xr-x        file_85.txt
        2048 1995-11-15 01:12:10 UTC -rwxr-xr-x        file_86.txt
          64 1978-11-02 02:42:57 UTC -rwxr-xr-x        file_87.txt
         512 2001-03-05 20:41:06 UTC -rwxr-xr-x        file_88.txt
        2048 1989-04-11 02:41:46 UTC -rwxr-xr-x        file_89.txt
          64 1979-09-06 06:02:03 UTC -rwxr-xr-x        file_9.txt
        1024 1995-10-01 19:12:03 UTC -rwxr-xr-x        file_90.txt
         512 1996-08-04 13:16:35 UTC -rwxr-xr-x        file_91.txt
          32 1978-08-27 07:58:57 UTC -rwxr-xr-x        file_92.txt
         512 1981-08-22 01:37:50 UTC -rwxr-xr-x        file_93.txt
        1024 1985-09-17 02:12:57 UTC -rwxr-xr-x        file_94.txt
        2048 1970-07-02 17:02:17 UTC -rwxr-xr-x        file_95.txt
           2 1978-01-16 23:36:35 UTC -rwxr-xr-x        file_96.txt
          16 1995-07-18 08:21:04 UTC -rwxr-xr-x        file_97.txt
         512 1975-04-11 23:31:11 UTC -rwxr-xr-x        file_98.txt
           2 1999-11-05 02:20:51 UTC -rwxr-xr-x        file_99.txt
